cmd/collector/app/span_processor_test.go

```diff
@@ -333,17 +333,22 @@ func TestSpanProcessorWithNilProcess(t *testing.T) {
 }
 
 func TestSpanProcessorWithCollectorTags(t *testing.T) {
-
 	testCollectorTags := map[string]string{
-		"extra": "tag",
+		"extra":     "tag",
+		"dedup-key": "dedup-value",
```

Comment:
Can you add the case for the other condition, where the key exists but the value isn't a match?

Reply:
Done